src/main/java/org/jenkins/tools/test/model/PluginRemoting.java

```diff
@@ -118,13 +118,7 @@ private String retrievePomContentFromInputStream(InputStream pluginUrlStream) th
                 zipEntry = zin.getNextEntry();
             }
 
-            StringBuilder sb = new StringBuilder();
-            byte[] buf = new byte[1024];
-            int n;
-            while ((n = zin.read(buf, 0, 1024)) > -1)
-                sb.append(new String(buf, 0, n, Charset.defaultCharset()));
-
-            return sb.toString();
```

Comment:
Unrelated code cleanup while I was here: using Java 11 functionality where possible. This should also be in UTF-8 because this is a POM file built by Maven by the plugin maintainer, not a file written in the current user's default character set.